site/app/views/PollView.php

```diff
@@ -90,11 +91,13 @@ public function pollForm(?Poll $poll = null) {
         $this->core->getOutput()->addVendorCss(FileUtils::joinPaths('flatpickr', 'flatpickr.min.css'));
         $this->core->getOutput()->addVendorJs(FileUtils::joinPaths('flatpickr', 'plugins', 'shortcutButtons', 'shortcut-buttons-flatpickr.min.js'));
         $this->core->getOutput()->addVendorCss(FileUtils::joinPaths('flatpickr', 'plugins', 'shortcutButtons', 'themes', 'light.min.css'));
+        $is_survey = $poll !== null ? $poll->isSurvey() : false;
```

Comment:
```suggestion
        $is_survey = $poll?->isSurvey() ?? false;
```

Reply:
using the null coalescing operator (??) and the null-safe operator (?->) to make it concise